go/oasis-node/cmd/debug/txsource/workload/queries.go

```diff
@@ -27,10 +28,17 @@ const (
 	// NameQueries is the name of the queries workload.
 	NameQueries = "queries"
 
+	// CfgNumKeptVersions is the number of last versions that the nodes are keeping (e.g., due to
```

Comment:
Maybe just note somewhere that runtime queries will still be done for all versions

Reply:
:+1: